src/main/java/io/javarig/config/DefaultConfigValues.java

```diff
@@ -3,4 +3,5 @@
 public class DefaultConfigValues {
     public final static int DEFAULT_MAX_SIZE_EXCLUSIVE = 15;
     public final static int DEFAULT_MIN_SIZE_INCLUSIVE = 5;
+    public final static String DEFAULT_REGEX_PATTERN = "[a-zA-Z0-9]*";
```

Comment:
As a default behaviour we don't want to limit string values to just alphanumerics, I think in case of no pattern was provided we should let it null, and generate it the same way we did before.

Reply:
that literally was the default behavior xd😆  (check https://github.com/oubaydos/javaRIG/blob/main/src/main/java/io/javarig/generator/StringGenerator.java#L19)

Reply:
So it was wrong in the first place 😂
What do you think ? I don't think we will always need alphanumerics